pymonetdb/mapi.py

```diff
@@ -373,3 +398,21 @@ def set_reply_size(self, size):
         """
 
         self.cmd("Xreply_size %s" % size)
+
+
+class HandshakeOption:
```

Comment:
i think this is good as it is, but dataclasses and/or named tuple are ideals for this kind of classes.

https://docs.python.org/3/library/dataclasses.html

Reply:
good idea, I'll make it a dataclass

Reply:
on second thought, dataclasses are only supported in Python 3.7, and pymonetdb also supports 3.6 doesn't it?

Reply:
true, good point. Then I would use a named tuple, which is practically the same thing

https://docs.python.org/3/library/collections.html#collections.namedtuple

Reply:
Named tuples are immutable so I can't set the .sent field. I'll mention `@dataclass` in a comment above the class so it's clear that it contains only data and no behavior. When 3.6 goes out of support we can make it a proper `@dataclass`.